$MirOS: ports/www/dillo/stable/patches/patch-src_IO_dpi_c,v 1.2 2005/06/19 17:20:19 bsiegert Exp $
--- src/IO/dpi.c.orig	Wed Mar 15 15:49:41 2006
+++ src/IO/dpi.c	Mon May 15 17:49:33 2006
@@ -328,7 +328,7 @@ static gint Dpi_start_dpid(void)
    pid = fork();
    if (pid == 0) {
       /* This is the child process.  Execute the command. */
-      gchar *path1 = a_Misc_prepend_user_home(".dillo/dpid");
+      gchar *path1 = a_Misc_prepend_user_home(".etc/dillo/dpid");
       Dpi_close_fd(st_pipe[0]);
       if (execl(path1, "dpid", NULL) == -1) {
          g_free(path1);
@@ -401,7 +401,7 @@ static gchar *Dpi_get_dpid_uds_dir(void)
    gchar dpid_uds_dir[256], *p;
 
    saved_name_filename =
-      g_strconcat(g_get_home_dir(), "/", ".dillo/dpi_socket_dir", NULL);
+      g_strconcat(g_get_home_dir(), "/", ".etc/dillo/dpi_socket_dir", NULL);
    in = fopen(saved_name_filename, "r");
    g_free(saved_name_filename);
 
